src/SerialPort.ts

```diff
@@ -64,7 +65,7 @@ class SerialPort {
    * @param {string} hex the hex of data
    * @returns {Promise} success promise
    */
-  send(hex: string) {
+  send(hex: string): boolean {
```

Comment:
```suggestion
  send(hex: string): Promise<boolean> {
```

Reply:
Is that a Promise?

Reply:
Yes, I've missed the Java implementation. It's a promise.